tests/SonataDoctrineMongoDBAdminBundleTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineMongoDBAdminBundle\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\DoctrineMongoDBAdminBundle\DependencyInjection\Compiler\AddGuesserCompilerPass;
+use Sonata\DoctrineMongoDBAdminBundle\DependencyInjection\Compiler\AddTemplatesCompilerPass;
+use Sonata\DoctrineMongoDBAdminBundle\SonataDoctrineMongoDBAdminBundle;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @author Marko Kunic <kunicmarko20@gmail.com>
```

Comment:
I dont think he should be the author of this files, even if you copy it from the doctrine orm, wdyt?